content/pages/healthcare/apply.md

```diff
@@ -66,7 +66,6 @@ To update your personal, insurance, or financial information from a previous app
 ### Additional forms for caregivers, National Guard, and Reserve Servicemembers
```

Comment:
Because there doesn't exist a different way for National Guard to apply, unfortunately. It's something we're trying to figure out, but the current application is super broken and was never officially approved. So we should just remove reference to them in those two bullets and just include something about caregivers.